com.b2international.snomed.ecl.ui/src/com/b2international/snomed/ecl/ui/EclTokens.java

```diff
@@ -17,45 +17,80 @@
 
 import java.util.HashSet;
 import java.util.Set;
-
 /**
+ * Tokens are made from the rules in the Ecl.xtext
  * @since 1.3.0
  */
 public class EclTokens {
 
 	public static final String PREFIX = "RULE_";
+	
+	public static final String SNOMED_IDENTIFIER = "SnomedIdentifier";
+	public static final String PIPE = "PIPE";
```

Comment:
Would it be possible to reuse these from somewhere? I think Xtext generates these tokens already to a class.

Reply:
Done